core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java

```diff
@@ -28,7 +30,7 @@
    *
    * @return a map of statistics per name
    */
-  Map<String, TypedValueStatistic> getKnownStatistics();
+  Map<String, ValueStatistic<?>> getKnownStatistics();
```

Comment:
It's ok for this case because the type depends on the `StatisticType` returned by `ValueStatistic ` so this is up to the user to know if the type is a string, long, int, double, etc based on the stat name and type.

in the management layer, I also have a call like this 

`public <T extends Serializable> Optional<T> getStatistic(String name);`

The downside is that it forces sometimes the user to put the expected type in the call :-(